### PR TITLE
Add Users API

### LINKED ISSUE #26: Add Users API
API: https://docs.dev.to/api/#tag/users

> For implementing it, you can use my `ArticleService` class as a guide and create an appropriate interface like how I have `IArticleApi`. The `ApiService` class provides the helper methods for constructing the requests for GET and PUT.

### PR DESCRIPTION
**Pull request by fgiuliani**
closes #26

### CODE REVIEW

File: src/DevTo.Api/UsersService.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	public class UsersService : ApiService, IUsersApi
+	{
+		public UsersService(Uri baseUri, HttpClient httpClient) : base(baseUri, httpClient) { }
+
+		public Task<User> GetUserAsync(int id) => GetAsync<User>($"/api/users/{id}");
+
+		public Task<User> GetUserAsync(string username)
+			=> GetAsync<User>("/api/users/by_username", new Dictionary<string, object>
+				{
+				  { "url", username },
```
Comment (Turnerj) [inline src/DevTo.Api/UsersService.cs:18]: I'm not sure if the parameter is actually "url" though I also can't double check at the moment as the DEV API docs seem to be down.

**Comment by Turnerj** [comment]
Looks good to me, just one query regarding the parameter for retrieving users but unfortunately I can't double check at the moment as the docs seem to be down for me.

**Comment by fgiuliani** [comment]
@Turnerj this is what the API doc details

Screen Shot 2020-06-13 at 10 15 19

**Comment by Turnerj** [comment]
OK, awesome. Its a bit surprising that's the parameter name they used but if that's what the docs say, it's what we will set!

**Pull request merged**